### ISSUE TITLE
how to setup this on ec2 ?

### ISSUE THREAD

**Issue by d1pakda5** [olacabs/jackhammer#51]
i have done the proper setup on ec2 .

and port are open too all inbound and outbound but still not working .

**Comment by kmadhusudhan** [comment]
if any domain configured to EC2 then you can access with domainname:5000. if you want configured  different port ,you can change in  docker-compose..yml

**Comment by d1pakda5** [comment]
@KMadhuSudhan  I just tried checking the port to 80 but still no success . in my security group port 80 and 5000 also opened

**Comment by kmadhusudhan** [comment]
are you able to access other applications from EC2 ?

**Comment by d1pakda5** [comment]
yes on same port i verified running another service . its working but for jackhammer its not working

**Comment by kmadhusudhan** [comment]
can please confirm that client is running ? . login to docker image and  check for rails process
